official/benchmark/benchmark_uploader_main.py

```diff
@@ -54,6 +54,10 @@ def main(_):
   uploader.upload_metric_file(
```

Comment:
It would be nice if we did a three part status-- Running, Success, Failure. Before running, mark running; update to success on success, or failure on failure.